pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -448,6 +448,21 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		},
 		{
 			ObjectMeta: kapi.ObjectMeta{
+				Name: ImageSignerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("get"),
```

Comment:
This won't work right because these are cluster scoped resources and this role is granted at a project scope.